website/docs/business-case/BusinessCaseIndustry.md

```diff
@@ -0,0 +1,151 @@
+
+#### Access to Trade Finance
+The Asian Development Bank (ADB) estimates that the global trade finance gap was approximately $2.5 trillion in 2022, up from $1.5 trillions in 2016 with a significant portion attributable to SMEs applicants, lack of visibility, and issues with country risk, credit-worthiness and lack of sufficient information by the applicant. At the same time, Supply Chain Finance (SCF) has grown from $330 billion in 2015 to $1.8 trillion in 2021, despite this growth, SCF has has not yet had a major impact in reducing the trade finance gap due to difficulty reaching past tier 1 suppliers. By adopting the UNTP, this gap can be reduced by enabling more companies to access preferential financing thanks to increased visibility over ESG credentials and ability to provide identity assurance from a trusted register, combined with SCF reverse factoring operating models which reduce applicants risk by tying the financing to the buyer credit risk.
+
+* *References* Asian Development Bank (ADB), [Trade Finance Gaps Growth and Jobs Survey 2021](https://www.adb.org/publications/2021-trade-finance-gaps-growth-jobs-survey), 
+ [Trade finance gaps growth jobs survey 2023](https://www.adb.org/publications/2023-trade-finance-gaps-growth-jobs-survey). [Deep-Tier Supply Chain Finance 2022](https://www.adb.org/publications/deep-tier-supply-chain-finance)
+
+#### Reduced Finance Costs
+
+According to the International Finance Corporation (IFC), companies that adopt sustainable practices can reduce their financing costs by up to 20% due to lower risk premiums and better access to capital.
+
+* *References* International Finance Corporation (IFC), "Sustainable Finance: Creating Value for Companies and Investors," 2020.
+
+#### Improved margins
+
+A study by the Global Reporting Initiative (GRI) found that companies with strong ESG performance can achieve up to a 10% improvement in profit margins due to enhanced operational efficiencies and lower financing costs.
+
+* *References* Global Reporting Initiative (GRI), "The Business Case for ESG: How Sustainability Can Drive Financial Performance," 2019.
+
+#### Cost of Goods Sold
+A report by McKinsey & Company indicates that companies with optimized supply chain financing can reduce their cost of goods sold by 5% to 10% due to lower financing costs and improved supply chain efficiencies.
+
+* *References* McKinsey & Company, "Supply Chain Finance: The Key to Unlocking Working Capital," 2019.
```

Comment:
Also for this one nothing to be found on the internet or Mckinsey website. Perhaps who has put these references, has access to some private databases, PDFs or something like that? Could you put where these are coming from?